protocol/revapi.json

```diff
@@ -336,6 +336,54 @@
           "code": "java.method.removed",
           "old":  "method io.camunda.zeebe.protocol.record.value.ImmutableCompensationSubscriptionRecordValue io.camunda.zeebe.protocol.record.value.ImmutableCompensationSubscriptionRecordValue::withCompensableActivityScopeId(long)",
           "justification": "This functionality is not yet released"
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old":  "method io.camunda.zeebe.protocol.record.value.ImmutableUserTaskRecordValue.Builder io.camunda.zeebe.protocol.record.value.ImmutableUserTaskRecordValue.Builder::withCandidateGroups(java.lang.String)",
+          "justification": "This functionality is not yet released"
+        },
```

Comment:
❌ Please add such ignores to `ignored-changes.json` so they can be cleared on release: https://github.com/camunda/zeebe/wiki/Breaking-Changes#java

It could also benefit from a more thorough justification, perhaps linking your analysis of the breaking change

Reply:
Thanks, @korthout, I wasn't aware of those docs, glad we have them 👍 
I'll add this 🙂

Reply:
Just be certain: we add such changes twice now, in the `revapi.json` and also in an `ignored-changes.json` file?

Reply:
Addressed with https://github.com/camunda/zeebe/pull/16141/commits/cc195fff780ceb3500cc637993548ec5e2c8c4fa

Reply:
>we add such changes twice now, in the revapi.json and also in an ignored-changes.json file?

@tmetzke No, only in the `ignored-changes.json` file. The `revapi.json` file is for long term configurations, while the ignored changes are cleared with every release.

Reply:
Thanks, @korthout, that makes a lot of sense. 👍 
Looking at [this PR](https://github.com/camunda/zeebe/pull/13068) as an example (there are more like it), we added similar entries only to the `revapi,json`. We did not create the `ignored-changes.json` or add entries there. Thus I am wondering if we either
1. Need to make the team aware again of how this is done.
2. Need to adjust the documentation and/or processes to reflect how we do this now.

Reply:
That's a good idea @tmetzke. Let's check first if there is some reason I'm missing.

@Zelldon do you remember why [these revapi ignores](https://github.com/camunda/zeebe/pull/13068/files#diff-92a55e36a54aca257697d944be71f135bc5e2644a351e822545283a1df1652deR186) were added to `revapi.json` rather than `ignored-changes.json`?